### PR TITLE
storage account validation

### PR DESCRIPTION
**Pull request by prekoa**

### CODE REVIEW

File: internal/providers/azure/objectstore.go [hunk 1]
```diff
@@ -164,6 +168,12 @@ func (s *ObjectStore) getLogger() logrus.FieldLogger {
 	})
 }
 
+func isValidStorageAccount(storageAccount string) bool {
```
Comment (lpuskas) [inline internal/providers/azure/objectstore.go]: reaname it to isValidStorageAccountName, or simply isValid and add godoc explayning what "valid" means eventually a link to the regexp or the description of a valid storage account name

Comment (prekoa) [reply internal/providers/azure/objectstore.go]: thx, fixed

File: internal/providers/azure/objectstore.go [hunk 2]
```diff
@@ -164,6 +168,13 @@ func (s *ObjectStore) getLogger() logrus.FieldLogger {
 	})
 }
 
+// https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-manager-storage-account-name-errors
+func isValidStorageAccountName(storageAccount string) bool {
+	match, _ := regexp.MatchString(`^[a-z0-9]{3,24}$`, storageAccount)
```
Comment (stoader) [inline internal/providers/azure/objectstore.go]: This regex could be made as a const.

Comment (prekoa) [reply internal/providers/azure/objectstore.go]: fixed, thx

**Pull request merged**